templates/docs/examples/patterns/navigation/deprecated.html

```diff
@@ -0,0 +1,67 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Navigation / Deprecated{% endblock %}
```

Comment:
We should have a dark version of the deprecated example as well. But maybe instead of adding separate file for it just add it to this one as a second component below.

Reply:
I included them both in the example already: 

![Screenshot 2022-03-29 at 12 05 42](https://user-images.githubusercontent.com/58959073/160598141-e4531901-6b0e-40d6-9ac1-f1e1ced567c5.png)